omod/src/test/resources/TagResourceDataSet.xml

```diff
@@ -0,0 +1,29 @@
+<dataset>
```

Comment:
This is redundant, you should be using the same test dataset, if you need to inject extra test data you only add just the extra rows, I can see any reason why you would need to have different datasets.

You should be able to access the test dataset from the omod project because the api is a dependency of the omod, you might want to check that your maven setup is correct